ha/setup/ha_setup.py

```diff
@@ -855,6 +855,16 @@ def main(argv: list):
             log_path = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}path")
             log_level = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}level")
             Log.init(service_name='ha_setup', log_path=log_path, level=log_level)
+        elif sys.argv[1] == "cleanup":
+            if not os.path.exists('const.HA_CONFIG_FILE'):
+                a_str = f'Cleanup can not be proceed as \
+                           HA config file: {const.HA_CONFIG_FILE} \
+                           is missing. Either cleanp is already done or there \
+                           is some other problem'
+                Log.error(f'{a_str}')
```

Comment:
Will this log work in all cases?

Reply:
Didn't get your question actually.
This will be logged only in cleanup case where cleanup is requested 2nd time (where conf file is missing).

Reply:
I mean will the logging work here in this case?

Reply:
I got it now. We can't use this Log here. Because it will not be initialized here. Hence removing now.